README.md

```diff
@@ -31,11 +32,11 @@ Installation
 
 Use go get.
 
-	go get github.com/go-playground/validator/v10
```

Comment:
The indentation was on purpose to render the command as code. Not it is displayed as standard text.